docs/ops/deploy.md

```diff
@@ -25,9 +25,27 @@ Edit conf/titan.toml and set the pd-addrs
 pd-addrs="tikv://your-pd-addrs:port"
 ```
 
+### Enable Multi-tenancy
+First, set auth value in the 'server' section of conf/titan.toml
+
+```
+auth = "somepwd"
+```
+
+Then，use ./tools/token/token to gen a client token.
```

Comment:
may use 'generate' instead of 'gen' is better?

Reply:
And no comma for 'Then'

Reply:
thanks, fixed.